### PR TITLE
Pressure sore Component of Critical Care 1404

### PR DESCRIPTION
**Pull request by peeyush14goyal**
- [x] 1. Front View and Back View Diagrams Added
- [x] 2. Multi-select on diagrams
- [x] 3. Sliders for Risk Severity Scale for both diagrams
- [x] 4. Label appear on hover of any part of Diagram

### CODE REVIEW

File: src/Components/CriticalCareRecording/types/CriticalCare__PressureSore.res [hunk 1]
```diff
@@ -0,0 +1,15 @@
+type typ = {d: string, transform: string, label: string}
```
Comment (bodhish) [inline src/Components/CriticalCareRecording/types/CriticalCare__PressureSore.res]: Name types

File: src/Components/CriticalCareRecording/types/CriticalCare__PressureSore.res [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+let d = typ => typ.d
+let transform = typ => typ.transform
+let label = typ => typ.label
+
+type ty = {
```
Comment (bodhish) [inline src/Components/CriticalCareRecording/types/CriticalCare__PressureSore.res]: Use `t` insted of `ty`

File: src/Components/Patient/ManagePatients.tsx [hunk 3]
```diff
@@ -295,12 +291,10 @@ export const PatientManager = (props: any) => {
 
   const fetchFacilityName = useCallback(
     async (status: statusType) => {
-      setIsLoading(true);
```
Comment (bodhish) [inline src/Components/Patient/ManagePatients.tsx:298]: Why do we have these changes?

**Review by bodhish — CHANGES_REQUESTED** [review]

File: src/Components/Patient/ManagePatients.tsx (hunk 3 above)
Comment (peeyush14goyal) [reply src/Components/Patient/ManagePatients.tsx:298]: I was initially working on `issue-1404` branch and there is a commit by Sandra so I suppose these are those changes

File: src/Components/CriticalCareRecording/types/CriticalCare__PressureSore.res (hunk 2 above)
Comment (peeyush14goyal) [reply src/Components/CriticalCareRecording/types/CriticalCare__PressureSore.res]: Done

File: src/Components/CriticalCareRecording/types/CriticalCare__PressureSore.res (hunk 1 above)
Comment (peeyush14goyal) [reply src/Components/CriticalCareRecording/types/CriticalCare__PressureSore.res]: `typ` changed to `type_for_path`

**Pull request merged**